azure/inputseeders/azure/id.ftl

```diff
@@ -0,0 +1,139 @@
+[#ftl]
+
+[@addInputSeeder
+    id=AZURE_INPUT_SEEDER
+    description="Azure provider inputs"
+/]
+
+[@addSeederToInputStage
+    inputStage=MASTERDATA_SHARED_INPUT_STAGE
+    inputSeeder=AZURE_INPUT_SEEDER
+/]
+
+[@addSeederToInputStage
+    inputStage=MOCK_SHARED_INPUT_STAGE
+    inputSeeder=AZURE_INPUT_SEEDER
+/]
+
+[@addSeederToInputStage
+    inputSources=[MOCK_SHARED_INPUT_SOURCE]
+    inputStage=COMMANDLINEOPTIONS_SHARED_INPUT_STAGE
+    inputSeeder=AZURE_INPUT_SEEDER
+
+/]
+
+[#macro azure_input_loader path]
+    [#assign azure_cmdb_regions =
+        (
+            getPluginTree(
+                path,
+                {
+                    "AddStartingWildcard" : false,
+                    "AddEndingWildcard" : false,
+                    "MinDepth" : 1,
+                    "MaxDepth" : 1,
+                    "FilenameGlob" : "regions.json"
+                }
+            )[0].ContentsAsJSON
+        )!{}
+    ]
+    [#assign azure_cmdb_masterdata =
+        (
+            getPluginTree(
+                path,
+                {
+                    "AddStartingWildcard" : false,
+                    "AddEndingWildcard" : false,
+                    "MinDepth" : 1,
+                    "MaxDepth" : 1,
+                    "FilenameGlob" : "masterdata.json"
+                }
+            )[0].ContentsAsJSON
+        )!{}
+    ]
+[/#macro]
+
+[#function azure_input_masterdata_seeder filter state]
```

Comment:
MIght be better as 

`azure_inputseeder_masterdata`

Reply:
done